cri/v1alpha2/cri_utils.go

```diff
@@ -610,6 +616,19 @@ func applyContainerSecurityContext(lc *runtime.LinuxContainerConfig, podSandboxI
 
 // Apply Linux-specific options if applicable.
 func (c *CriManager) updateCreateConfig(createConfig *apitypes.ContainerCreateConfig, config *runtime.ContainerConfig, sandboxConfig *runtime.PodSandboxConfig, podSandboxID string) error {
+	// Apply runtime options.
+	if annotations := config.GetAnnotations(); annotations != nil {
+		createConfig.HostConfig.Runtime = annotations[anno.KubernetesRuntime]
```

Comment:
if the `annotations[anno.KubernetesRuntime]` is emtpy?

Reply:
`createConfig.HostConfig.Runtime` will be `""`, and then PouchContainer will use the `DefaultRuntime`,which is runc.
```
// set container runtime
if config.HostConfig.Runtime == "" {
	config.HostConfig.Runtime = mgr.Config.DefaultRuntime
}
```